README.rst

```diff
@@ -95,7 +95,7 @@ Symfony is smart enough to reverse those tasks when uninstalling and
 unconfiguring the dependencies.
 
 There are several types of tasks, which are called **configurators**:
-``copy-from-recipe``, ``copy-from-package``, ``bundles``, ``env``,
+``copy-from-recipe``, ``copy-from-package``, ``bundles``, ``env``, ``container``
```

Comment:
Maybe we should just remove this list - does it add value when they are listed right below? I didn’t realize this was here - so it’s also missing the new add-lines configurator